specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -1144,6 +1144,230 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration Config successfully created",
+            "schema": {
+              "$ref": "#/definitions/MigrationConfigProperties"
+            }
+          },
+          "201": {
+            "description": "Migration Config creation request received"
+          },
+          "default": {
+            "description": "ServiceBus error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "migrationConfigurations"
+        ],
+        "operationId": "migrationConfigurations_Delete",
+        "description": "Deletes an MigrationConfig",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete Migration Config request accepted"
+          },
+          "default": {
+            "description": "ServiceBus error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "migrationConfigurations"
+        ],
+        "operationId": "migrationConfigurations_Get",
+        "description": "Retrieves Migration Config",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved Migration Config)",
+            "schema": {
+              "$ref": "#/definitions/MigrationConfigProperties"
+            }
+          },
+          "default": {
+            "description": "ServiceBus error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/migrationConfigurations/$default/upgrade": {
+      "post": {
+        "tags": [
+          "migrationConfigurations"
+        ],
+        "operationId": "MigrationConfig_upgrade",
+        "description": "This operation disables the Migration",
```

Comment:
This should be named as "migrationConfigurations_DisableMigration" or something more self explanatory. No need to move it to a separate operation group on the client SDKs;.

Reply:
changed it to 'migrationConfigurations_CompleteMigration' and changed the description. as this operation will complete migration process